src/waveform/renderers/waveformmark.cpp

```diff
@@ -5,21 +5,23 @@
 
 #include "waveformmark.h"
 
-WaveformMark::WaveformMark(int hotCue)
+WaveformMark::WaveformMark( const QString& group,
+                            const QDomNode& node,
+                            const SkinContext& context,
+                            const WaveformSignalColors& signalColors,
+                            int hotCue,
+                            QString item)
     : m_iHotCue(hotCue) {
-}
-
-void WaveformMark::reset(int hotCue) {
-    WaveformMark(hotCue).swap(*this);
-}
-
-void WaveformMark::setup(const QString& group, const QDomNode& node,
-                         const SkinContext& context,
-                         const WaveformSignalColors& signalColors) {
-    QString item = context.selectString(node, "Control");
+    if(item.isEmpty())
```

Comment:
Please always enclose if/else branches and loop bodies in curly braces, even if it is only a single line. Just by looking at the following lines of code you should notice that your code style is inconsistent with the existing code style ;)